en/08_Changelogs/6.2.0.md

```diff
@@ -23,6 +23,18 @@ title: 6.2.0 (unreleased)
 - In the `silverstripe/linkfield` module, the `.link-picker__delete` element has been moved up a level so that it is now a child of `.link-picker__link` and its tag has changed from `span` to `button`.
 - The viewport meta tag no longer includes `maximum-scale=1.0`, which prevented users from zooming in on mobile devices. It now includes `initial-scale=1.0` instead.
 
+#### Changes to usage of `font-icon-*` CSS classes
+
+The `font-icon-*` CSS classes used to add icons through the CMS use a font to render normal characters as icons.
+
+This is very useful for users who can see, but represents a problem for screen reader users, since screen readers want to read the icons like normal characters. For example, the `font-icon-search` icon uses the character "s", so screen readers will read out "s" whenever they encounter that icon.
```

Comment:
```suggestion
This is very useful for sighted users, but represents a problem for screen reader users, since screen readers want to read the icons like normal characters. For example, the `font-icon-search` icon uses the character "s", so screen readers will read out "s" whenever they encounter that icon.
```

Reply:
Done